src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/production/mega/GregTechMetaTileEntity_MegaAlloyBlastSmelter.java

```diff
@@ -366,14 +361,10 @@ public boolean checkRecipe(ItemStack aStack) {
 
     protected boolean processRecipe(ItemStack[] tItems, FluidStack[] tFluids) {
         if (tItems.length <= 0 && tFluids.length <= 0) return false;
-        long tVoltage = GT_ExoticEnergyInputHelper.getMaxInputVoltageMulti(getExoticAndNormalEnergyHatchList());
-        long tAmps = GT_ExoticEnergyInputHelper.getMaxInputAmpsMulti(getExoticAndNormalEnergyHatchList());
+        long tVoltage = this.getMaxInputVoltage();
+        long tAmps = (long) (this.getMaxInputAmps() * 0.8D); // Use 80% of the available amps to get working amps
```

Comment:
Normal energy hatches are also accepted in the structure, that should be checked